wdpk/zerotier/start.sh

```diff
@@ -1,4 +1,7 @@
-#! /bin/sh
-[ -f /tmp/debug_apkg ] && echo "APKG_DEBUG: $0 $@" >> /tmp/debug_apkg
+#!/bin/bash
+[ -f /tmp/debug_apkg ] && echo "APKG_DEBUG: $0 $@ $PWD" >> /tmp/debug_apkg
 
-/opt/bin/zerotier-one -d
\ No newline at end of file
+export ZEROTIER_HOME=/mnt/HD/HD_a2/.systemfile/zerotier
```

Comment:
Probably the best way to manage package data is to leave it inside `Nas_Prog/zerotier`, but move it here to `.systemfile` in `preinst.sh`. This is the pre-upgrade hook that runs before `remove.sh`.
Then in `install.sh`, restore the data from `.systemfile`

Reply:
I saw that on OS5 but these files never change and losing them can be a problem as you lose access to the vpn, do you prefer me to do that logic or just leave the files on systemfiles as they never actually get upgraded?